### PR TITLE
OOC: Sniblet: you cant hug a corpse

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

untill now!

#### Changelog

:cl:  
fix: corpses can now be hugged.
/:cl:

### CODE REVIEW

**Comment by thehighwayman** [comment]
Only way someone will let nich hug them

**Comment by GuppyLaxx** [comment]
you forgot the sound

**Comment by ShadowDeath6** [comment]
Ehh I would prefer it to be trying to shake them up (like if they were unconscious) while displaying the no cpr on dead people message

File: yogstation/code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/mob/living/carbon/human/do_cpr(mob/living/carbon/C)
+	CHECK_DNA_AND_SPECIES(C)
+
+	if(C.stat == DEAD || (C.has_trait(TRAIT_FAKEDEATH)))
+		visible_message("<span class='notice'>[src] hugs [C] to make [C.p_them()] feel better!</span>", \
+			"<span class='notice'>You hug [C] to make [C.p_them()] feel better!</span>")
+		return
+
+	.=..()
```
Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [inline yogstation/code/modules/mob/living/carbon/human/human.dm]: `.=..()` on last line :100:

**Pull request closed without merging**